stored_requests/events/database/database.go

```diff
@@ -113,13 +114,18 @@ func (e *PostgresEventProducer) fetchAll() (fetchErr error) {
 	return nil
 }
 
-func (e *PostgresEventProducer) fetchDelta() (fetchErr error) {
+func (e *DatabaseEventProducer) fetchDelta() (fetchErr error) {
 	timeout := e.cfg.CacheUpdateTimeout * time.Millisecond
 	ctx, cancel := context.WithTimeout(context.Background(), timeout)
 	defer cancel()
 
 	startTime := e.time.Now().UTC()
-	rows, err := e.cfg.DB.QueryContext(ctx, e.cfg.CacheUpdateQuery, e.lastUpdate)
+
+	params := []db_provider.QueryParam{
+		{Name: "LAST_UPDATE", Value: e.lastUpdate},
```

Comment:
Should this be `LAST_UPDATED` instead of `LAST_UPDATE`?

Reply:
Fixed.